oshdb-util/src/main/java/org/heigit/ohsome/oshdb/util/mappable/OSMContribution.java

```diff
@@ -83,24 +83,18 @@ public interface OSMContribution extends OSHDBMapReducible, Comparable<OSMContri
   OSHEntity getOSHEntity();
 
   /**
-   * Checks if this contribution is of the given contribution type.
+   * Checks if this contribution is of the given contribution type,
    *
    * <p>
-   * It can be one or more of:
-   * </p>
-   * <ul>
-   *   <li>CREATION</li>
-   *   <li>DELETION</li>
-   *   <li>TAG_CHANGE</li>
-   *   <li>GEOMETRY_CHANGE</li>
-   * </ul>
-   *
-   * <p>
-   * If this is a entity creation or deletion, the other flags are not set (even though one might
-   * argue that a just created object clearly has a different geometry than before, for example).
-   * </p>
-   *
-   * @return a set of modification type this contribution made on the underlying data
+   * i.e. checks if the given contribution type is in the
+   * {@code EnumSet<ContributionType>} returned by
+   * {@link #mgetContributionTypes() getContributionTypes}. This method is
```

Comment:
is the extra `m` a typo?

```suggestion
   * {@link #getContributionTypes() getContributionTypes}. This method is
```